src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -106,15 +106,17 @@ def _get_reductions_list(self, reduction_type):
                             result.append(arg.name)
         return result
 
-class OMPSingleDirective(OMPDirective):
 
+class OMPSingleDirective(OMPDirective):
     '''
     Class representing an OpenMP SINGLE directive in the PSyclone AST.
-    :param list children: list of Nodes that are children of this Node.
-    :param parent: the Node in the AST that has this directive as a child.
+
```

Comment:
Not your fault (since you've just copied this) but this was before we named the "PSyIR". Therefore please s/PSyclone AST/PSyIR/

Reply:
Done and fixed for the location I copied from also.

Reply:
Thanks :-)